autopush/tests/test_websocket.py

```diff
@@ -435,10 +423,10 @@ def test_close_with_delivery_cleanup_and_get_no_result(self):
 
         # Close the connection
         self.proto.onClose(True, None, None)
-        yield self._wait_for(lambda: len(self.metrics.mock_calls) > 2)
-        eq_(len(self.metrics.mock_calls), 3)
-        self.metrics.increment.assert_called_with(
-            "client.notify_uaid_failure", tags=None)
+        yield self._wait_for(lambda: len(self.metrics.mock_calls) > 0)
+        eq_(self.metrics.timing.call_args[0][0], 'ua.connection.lifespan')
+        # Wait for final cleanup (no error or metric produced)
+        yield sleep(1)
```

Comment:
Hrm, this might be another bit that got messed up in the patch I had to do to catch up. And looks like `client.notify_uaid_failure` isn't listed in the table. The metric had previously been removed.